superset/assets/spec/javascripts/explorev2/components/FieldSetRow_spec.js

```diff
@@ -25,7 +20,6 @@ describe('FieldSetRow', () => {
   });
 
   it('renders a FieldSet for each item in fieldSets array', () => {
-    const length = defaultProps.fieldSets.length;
-    expect(wrapper.find(FieldSet)).to.have.lengthOf(length);
```

Comment:
why don't we want to assert how many `FieldSet` components are rendered anymore? i'm not really sure how this change is testing the `FieldSetRow` component.